openedx/features/caliper_tracking/tests/expected/edx.forum.thread.voted.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Liked",
+  "actor": {
+    "id": "http://localhost:18000/u/verified",
+    "name": "verified",
+    "type": "Person"
+  },
+  "eventTime": "2018-10-18T13:52:24.825Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36",
+      "course_id": "course-v1:edX+DemoX+Demo_Course",
+      "course_user_tags": {},
+      "event_source": "server",
+      "event_type": "edx.forum.thread.voted",
+      "host": "8433fad63981",
+      "ip": "172.18.0.1",
+      "org_id": "edX",
+      "page": null,
+      "path": "/courses/course-v1:edX+DemoX+Demo_Course/discussion/threads/5bc887e1977195008e000000/upvote",
+      "session": "c8840acdd1697ed86c4a942adfde334a",
+      "user_id": 8
+    }
+  },
+  "id": "urn:uuid:cd088ca7-c044-405c-bb41-0b2a8506f907",
+  "object": {
+    "extensions": {
+      "commentable_id": "i4x-edx-eiorguegnru-course-foobarbaz",
+      "id": "5bc887e1977195008e000000",
+      "target_username": "test",
+      "undo_vote": false,
+      "user_course_roles": [],
+      "user_forums_roles": ["Student"],
+      "vote_value": "up"
+    },
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/i4x-edx-eiorguegnru-course-foobarbaz/threads/5bc887e1977195008e000000",
+    "type": "Thread"
+  },
+  "referrer": {
+    "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/i4x-edx-eiorguegnru-course-foobarbaz/threads/5bc887e1977195008e000000",
+    "type": "WebPage"
+  },
+  "type": "Event"
```

Comment:
Is there no event more suitable here?

Reply:
It should be ForumEvent or ThreadEvent but there was no suitable action supported by those events.